src/psyclone/tests/test_files/dynamo0p3/26.7_mixed_precision_abstract_solver_vector.f90

```diff
@@ -29,29 +29,44 @@
 ! OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 ! OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ! -----------------------------------------------------------------------------
-! Authors: R. W. Ford and A. R. Porter, STFC Daresbury Lab
-! Modified: I. Kavcic, Met Office
+! Author: R. W. Ford STFC Daresbury Lab
+!
+! Example where the field is dereferenced from an abstract_vector_type
+! and therefore has no type information. The LFRic rules state that an
```

Comment:
Is this rule documented somewhere? Just to check that I understand: this is saying nothing about the underlying precision of the field_type?

Reply:
The rule was never documented and was the cause of errors for Tom so is no longer a rule (an exception is now raised). I've updated the description.